core_backend/migrations/versions/1ff08438751b_add_language_table.py

```diff
@@ -0,0 +1,59 @@
+"""add language table
+
+Revision ID: 1ff08438751b
+Revises: f269c75dbf69
+Create Date: 2024-03-18 18:38:29.579259
+
+"""
+
+from datetime import datetime
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = "1ff08438751b"
+down_revision: Union[str, None] = "f269c75dbf69"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "languages",
+        sa.Column("language_id", sa.Integer(), nullable=False),
+        sa.Column("language_name", sa.String(), nullable=False),
+        sa.Column("is_default", sa.Boolean(), nullable=False),
+        sa.Column("created_datetime_utc", sa.DateTime(), nullable=False),
+        sa.Column("updated_datetime_utc", sa.DateTime(), nullable=False),
+        sa.PrimaryKeyConstraint("language_id"),
+    )
+    op.create_index(
+        "ix_languages_is_default_true",
+        "languages",
+        ["is_default"],
+        unique=True,
+        postgresql_where=sa.text("is_default IS true"),
+    )
+    # add default language
+    date = str(datetime.now())
+    op.execute(
```

Comment:
Added this script to add english as a defaut language. This is because we want to separate the language frontend from this feature since we haven't created wireframes yet.